.editorconfig

```diff
@@ -34,3 +34,6 @@ indent_size = 2
 [CMakeLists.txt]
 indent_style = space
 indent_size = 2
+
+[*.lua]
+indent_style = tab
```

Comment:
This should already be covered by the `[*]` entry at the top (also, the majority of our Lua scripts don't have the `.lua` extension).